src/stories/containers/transparency-report/transparency-report.tsx

```diff
@@ -95,45 +93,43 @@ export const TransparencyReport = ({ coreUnits, coreUnit }: TransparencyReportPr
               margin: '32px 0',
             }}
           />
-          {tabsIndex === TRANSPARENCY_IDS_ENUM.ACTUALS && isEnabled('FEATURE_TRANSPARENCY_NEW_TABLE') && (
+          {tabsIndex === TRANSPARENCY_IDS_ENUM.ACTUALS && (
             <TransparencyActuals
               code={code}
               currentMonth={currentMonth}
               budgetStatements={coreUnit?.budgetStatements}
               longCode={longCode}
             />
           )}
-          {tabsIndex === TRANSPARENCY_IDS_ENUM.FORECAST && isEnabled('FEATURE_TRANSPARENCY_NEW_TABLE') && (
+          {tabsIndex === TRANSPARENCY_IDS_ENUM.FORECAST && (
             <TransparencyForecast
               currentMonth={currentMonth}
               budgetStatements={coreUnit?.budgetStatements}
               code={code}
               longCode={longCode}
             />
           )}
-          {tabsIndex === TRANSPARENCY_IDS_ENUM.MKR_VESTING &&
-            isEnabled('FEATURE_MKR_VESTING') &&
-            isEnabled('FEATURE_TRANSPARENCY_NEW_TABLE') && (
-              <TransparencyMkrVesting
-                currentMonth={currentMonth}
-                budgetStatements={coreUnit?.budgetStatements}
-                code={code}
-                longCode={longCode}
-              />
-            )}
-          {tabsIndex === TRANSPARENCY_IDS_ENUM.TRANSFER_REQUESTS && isEnabled('FEATURE_TRANSPARENCY_NEW_TABLE') && (
+          {tabsIndex === TRANSPARENCY_IDS_ENUM.MKR_VESTING && (
+            <TransparencyMkrVesting
+              currentMonth={currentMonth}
+              budgetStatements={coreUnit?.budgetStatements}
+              code={code}
+              longCode={longCode}
+            />
+          )}
+          {tabsIndex === TRANSPARENCY_IDS_ENUM.TRANSFER_REQUESTS && (
             <TransparencyTransferRequest
               currentMonth={currentMonth}
               budgetStatements={coreUnit?.budgetStatements}
               code={code}
               longCode={longCode}
             />
           )}
-          {tabsIndex === TRANSPARENCY_IDS_ENUM.AUDIT_REPORTS && isEnabled('FEATURE_AUDIT_REPORTS') && (
```

Comment:
this element should keep the feature flag check

Reply:
you are right, my bad